.github/actions/setup-gopath/action.yml

```diff
@@ -7,7 +7,7 @@ runs:
       shell: bash
       run: |
         export GOPATH=$(go env GOPATH)
-        export PROJECT=$GOPATH/src/github.com/uber/jaeger-client-go
+        export PROJECT=$GOPATH/src/github.com/jaegertracing/jaeger-client-go/v3
```

Comment:
Do we need to include `/v3` if we are changing the import path anyway?

Reply:
i guess not, would be nice to be able to trigger the github actions, so i can fix those errors :D i bet more than just 1 will come up :)